webapp/locales/en.json

```diff
@@ -74,7 +74,12 @@
       "name": "Download Data"
     },
     "delete": {
-      "name": "Delete Account"
+      "name": "Delete my User Account",
+      "countPosts": "Delete my {count} posts",
+      "countComments": "Delete my {count} comments",
+      "accountDescription": "Be aware that your Post and Comments are important to our community. If you still choose to delete those you have to mark them below.",
+      "accountWarning": "You <b>CAN'T MANAGE</b> and <b>CAN'T RECOVER</b> your Account, Posts... after deleting your account!",
+      "success": "Account successfully deleted"
```

Comment:
Use https://www.deepl.com/translator for German translations and you will get an extra translation check from me for free :wink: